demo.html

```diff
@@ -0,0 +1,394 @@
+<!DOCTYPE html>
+<html lang = "en">
```

Comment:
I am not familiar with this lang = "en" thing? I think the DOCTYPE html does the job